lab-carolina/index.js

```diff
@@ -0,0 +1,5 @@
+const principal = require('./lib/read.js');
+
+principal(function(data){
```

Comment:
What's the reasoning behind the function name? Not a big deal, but want to be sure you're considering naming.